src/Microsoft.ML.StandardLearners/Standard/MultiClass/Ova.cs

```diff
@@ -55,9 +57,43 @@ public sealed class Arguments : ArgumentsBase
             public bool UseProbabilities = true;
         }
 
+        /// <summary>
+        /// Legacy constructor that builds the <see cref="Ova"/> trainer supplying the base trainer to use, for the classification task
+        /// through the <see cref="Arguments"/>arguments.
+        /// Developers should instantiate OVA by supplying the trainer argument directly to the OVA constructor
+        /// using the other public constructor.
+        /// </summary>
+        /// <param name="env">The private <see cref="IHostEnvironment"/> for this estimator.</param>
+        /// <param name="args">The legacy <see cref="Arguments"/></param>
         public Ova(IHostEnvironment env, Arguments args)
             : base(env, args, LoadNameValue)
         {
+            _args = args;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of <see cref="Ova"/>.
+        /// </summary>
+        /// <param name="env">The <see cref="IHostEnvironment"/> instance.</param>
+        /// <param name="singleEstimator">An instance of the <see cref="BinaryPredictionTransformer{IModel}"/> used as the base predictor.</param>
+        /// <param name="calibrator">The <see cref="ICalibratorTrainer"/> used.</param>
+        /// <param name="labelColumn">The name of the label colum.</param>
+        /// <param name="imputeMissingLabelsAsNegative">Whether to treat missing labels as having negative labels, instead of keeping them missing.</param>
+        /// <param name="maxCalibrationExamples">Number of instances to train the calibrator.</param>
+        /// <param name="useProbabilities">Use probabilities (vs. raw outputs) to identify top-score category.</param>
+        public Ova(IHostEnvironment env, TScalarTrainer singleEstimator, string labelColumn = DefaultColumnNames.Label,
+            bool imputeMissingLabelsAsNegative = false, ICalibratorTrainer calibrator = null,
```

Comment:
>calibrator [](start = 75, length = 10)

note that the old default is not null

Reply:
I can't create the calibrator here, because i need to pass the env.. 
there is no regression, as i am restoring the Platt calibrator in the base MetaMulticlassTrainer constructor. Is it enough to comment about it in the parameter?



---
In reply to: [217233354](https://github.com/dotnet/machinelearning/pull/865#discussion_r217233354) [](ancestors = 217233354)

Reply:
that's correct

---
In reply to: [217479852](https://github.com/dotnet/machinelearning/pull/865#discussion_r217479852) [](ancestors = 217479852,217233354)